pallets/api/src/fungibles/mod.rs

```diff
@@ -179,7 +203,7 @@ pub mod pallet {
 		///
 		/// # Returns
 		/// The balance of the specified account, or an error if the operation fails.
-		pub fn balance_of(id: AssetIdOf<T>, owner: &AccountIdOf<T>) -> BalanceOf<T> {
+		pub(crate) fn balance_of(id: AssetIdOf<T>, owner: &AccountIdOf<T>) -> BalanceOf<T> {
```

Comment:
Do we even needs these functions anymore? It's boilerplate which can be replaced by moving the assets function call into the match of read_state

Reply:
I think when more functions get added this might be cleaner. Otherwise it will be a big match statement where the implementation of those state queries also could get more complex in stead of a one liner.

Reply:
Youve got 10 lines of code, including comments, for something that could be a one liner in the match. That's a pattern that requires a decent amount of extra work/reviews that are simply unnecessary right now. We need to get stuff finished and deployed, we get there by writing as little code as possible. This is all internal to the crate, it can easily all be refactored into separate functions like this if ever required later.

Reply:
Yes this makes sense, refactored.